calico-cloud/get-started/system-requirements.mdx

```diff
@@ -0,0 +1,47 @@
+---
+description: Review cluster requirements to connect to Calico Cloud.
+---
+
+# System requirements
+
+Before you connect your cluster to Calico Cloud, make sure your cluster meets the system requirements.
+Your cluster must already have a CNI installed before you can connect to Calico Cloud.
+
+<!--  This file is referenced in the manager UI and its location should not be changed with accounting for references from pendo/manager UI. A symlink so this file exists at install/system-requirements.  -->
+
+## Kubernetes distributions and CNIs
+
+Calico Cloud works with Kubernetes on self-provisioned infrastructure and on managed Kubernetes distributions.
+To use Calico Cloud for both networking and network policy, your cluster must have Calico Open Source installed before you connect to Calico Cloud.
+For most managed distributions, you can use the provider's CNI for networking and use Calico Cloud for network policy.
+
+| Distribution | CNI |
+| --- | --- |
+| Kubernetes on self-provisioned infrastructure | - Calico Open Source 3.26 and 3.27 |
```

Comment:
We weren't directing people to use old versions but saying if they had those they could be connected. Stating those old versions was in the spirit of allowing upgrades for clusters that had old installations. We don't want to turn away  users because their clusters are running old versions of Calico.

It is a difference of expecting people to set up new clusters to connect to Calico Cloud vs connecting clusters they already have, I feel like you're writing the docs from the Calico or Enterprise point of setting up a new cluster but Calico Cloud is around being able to connect existing clusters.